### DISCUSSION TITLE
对初始权重操作后返回，初始权重不变

### DISCUSSION THREAD

**Discussion by BlingYYa** [secretflow/secretflow#936] [General]
在获取权重后对其进行操作得到新权重U，将U赋予初始权重，初始权重并没有发生改变。                            
U = method(sf.reveal(client_params)[i])
sf.reveal(client_params)[i] = U

**Comment by wangyixianggogo** [comment]
#937 在这里解决

**Reply by BlingYYa** [reply]
谢谢！
